internal/container/podman.go

```diff
@@ -77,6 +79,21 @@ func (e *podmanImpl) ContainerRun(ctx context.Context, image string, flags *RunF
 				podmanOpts.Ports[entry.HostPort] = entry.ContainerPort
 			}
 		}
+		if flags.HealthCmd != nil {
```

Comment:
do we need to do this for docker as well?

Reply:
I wanted to avoid writing something among the lines of `if podman`. Since our abstractions are `startContainer` and not `startDeployment`, I couldn't move this code to the postman abstraction.

This code will also run for docker, but docker will find the healthcheck, so nothing will happen (I'm implementing the docker code right now, that's why I put the PR back in draft)